<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPatientWindow</class>
 <widget class="QMainWindow" name="NewPatientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lbl_date">
          <property name="text">
           <string>Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_time">
          <property name="text">
           <string>Time</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="lbl_currentuser">
          <property name="text">
           <string>Current User :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_currentusername">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spnbox_height">
            <property name="maximum">
             <double>300.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lnedt_lastname">
            <property name="placeholderText">
             <string>Last Name</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="lbl_examtype">
            <property name="text">
             <string>Exam Type</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="cmbox_examtype"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_height">
            <property name="text">
             <string>Height in cm</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lnedt_firstname">
            <property name="placeholderText">
             <string>First Name</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rdbtn_male">
            <property name="text">
             <string>Male</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QRadioButton" name="rdbtn_female">
            <property name="text">
             <string>Female</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="lbl_age">
            <property name="text">
             <string>Age</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QDateEdit" name="dateEdit_dob">
            <property name="displayFormat">
             <string>dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spnbox_weight">
            <property name="maximum">
             <double>600.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbl_weight">
            <property name="text">
             <string>Weight in kg</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item row="0" column="3">
           <widget class="QWidget" name="wdgt_user" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QPushButton" name="btn_addphysician">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_addrefphysician">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_adddepartment">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QWidget" name="wdgt_admin" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QPushButton" name="btn_deletephysician">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_deleterefphysician">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_deletedepartment">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="wdgt_cmbox" native="true">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QComboBox" name="cmbox_physician"/>
             </item>
             <item>
              <widget class="QComboBox" name="cmbox_refphysician"/>
             </item>
             <item>
              <widget class="QComboBox" name="cmbox_department"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QWidget" name="wdgt_lbl" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="lbl_physician">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Physician</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbl_refphysician">
               <property name="text">
                <string>Ref. Physician</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbl_department">
               <property name="text">
                <string>Department</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_reset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ok">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1152</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lnedt_firstname</tabstop>
  <tabstop>lnedt_lastname</tabstop>
  <tabstop>spnbox_height</tabstop>
  <tabstop>spnbox_weight</tabstop>
  <tabstop>rdbtn_male</tabstop>
  <tabstop>rdbtn_female</tabstop>
  <tabstop>dateEdit_dob</tabstop>
  <tabstop>cmbox_examtype</tabstop>
  <tabstop>cmbox_physician</tabstop>
  <tabstop>btn_addphysician</tabstop>
  <tabstop>cmbox_refphysician</tabstop>
  <tabstop>btn_addrefphysician</tabstop>
  <tabstop>cmbox_department</tabstop>
  <tabstop>btn_adddepartment</tabstop>
  <tabstop>btn_ok</tabstop>
  <tabstop>btn_reset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
